packages/video_player/video_player/lib/video_player.dart

```diff
@@ -519,6 +532,43 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
     await _videoPlayerPlatform.setVolume(_textureId, value.volume);
   }
 
+  /// Returns true if picture in picture is supported on the device.
+  Future<bool> isPictureInPictureSupported() {
+    return _videoPlayerPlatform.isPictureInPictureSupported();
+  }
+
+  /// Prepare picture in picture by passing the location of the video player view
```

Comment:
Same comments here as in the platform interface about the division of API calls.